### PR TITLE
CLDC-3738: Sales - Update household characteristics Q29 and Q38 (25/26)

### PR DESCRIPTION
**Pull request by Dinssa**

### CODE REVIEW

File: app/models/form/sales/pages/buyer2_relationship_to_buyer1_yes_no.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      {
+        "joint_purchase?" => true,
+        "buyer_has_seen_privacy_notice?" => true,
+      },
+      {
+        "joint_purchase?" => true,
+        "buyer_not_interviewed?" => true,
+      },
+    ]
+  end
```
Comment (kosiakkatrina) [inline app/models/form/sales/pages/buyer2_relationship_to_buyer1_yes_no.rb]: I dont think `buyer_has_seen_privacy_notice` and `buyer_not_interviewed` is needed anymore if this page only exists on 2025. I think it's covered by privacy notice and interview questions being in the setup section

File: app/models/form/sales/pages/person_relationship_to_buyer_1_yes_no.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class Form::Sales::Pages::PersonRelationshipToBuyer1YesNo < ::Form::Sales::Pages::Person
```
Comment (kosiakkatrina) [inline app/models/form/sales/pages/person_relationship_to_buyer_1_yes_no.rb:1]: Can there be more than 1 partner?

File: app/models/form/sales/questions/person_relationship_to_buyer_1_yes_no.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      "value" => "Prefers not to say",
+    }]
+    @check_answers_card_number = person_index
+    @person_index = person_index
+    @question_number = question_number
+  end
+
+  def question_number
+    base_question_number = case form.start_date.year
+                           when 2023
```
Comment (kosiakkatrina) [inline app/models/form/sales/questions/person_relationship_to_buyer_1_yes_no.rb]: would it ever be 2023?

File: app/models/form/sales/pages/person_relationship_to_buyer_1_yes_no.rb (hunk 2 above)
Comment (Dinssa) [reply app/models/form/sales/pages/person_relationship_to_buyer_1_yes_no.rb:1]: It's possible, perhaps we'd want to introduce a soft validation when there are more than one.

File: app/models/form/sales/questions/person_relationship_to_buyer_1_yes_no.rb (hunk 3 above)
Comment (Dinssa) [reply app/models/form/sales/questions/person_relationship_to_buyer_1_yes_no.rb]: This is left over from when duplicating this question. I imagine we could update this when we have the right question number.

File: app/models/form/sales/pages/buyer2_relationship_to_buyer1_yes_no.rb (hunk 1 above)
Comment (Dinssa) [reply app/models/form/sales/pages/buyer2_relationship_to_buyer1_yes_no.rb]: Updated to depend just on joint_purchase? => true

File: app/models/form/sales/questions/person_relationship_to_buyer_1_yes_no.rb (hunk 3 above)
Comment (Dinssa) [reply app/models/form/sales/questions/person_relationship_to_buyer_1_yes_no.rb]: Updated this to more closely match how we're setting the question numbers elsewhere and just put a temp value for 2025 (same as 2024).

File: app/models/form/sales/pages/person_relationship_to_buyer_1_yes_no.rb (hunk 2 above)
Comment (kosiakkatrina) [reply app/models/form/sales/pages/person_relationship_to_buyer_1_yes_no.rb:1]: Just had a look and we already have one!

**Review by kosiakkatrina — APPROVED** [review]

**Pull request merged**